jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BuildFiles.java

```diff
@@ -98,4 +103,42 @@ public static JibContainerBuilder toJibContainerBuilder(
     }
     return containerBuilder;
   }
+
+  @VisibleForTesting
+  // TODO: add testing, need to do via intergration tests as there's no good way to extract out that
+  //   the base image was populated as the user intended currently.
+  static JibContainerBuilder createJibContainerBuilder(
```

Comment:
I think this can be private without @VisibleForTesting?

Reply:
oh right.